app/components/slack-context-menu.js

```diff
@@ -108,17 +108,28 @@
 		}));
 	}
 
-	function buildLinkMenu(menu, target) {
+	function buildLinkMenu(menu, target, teamUrl) {
 		if (target.attributes.href && target.attributes.href.value) {
 			var hrefValue = target.attributes.href.value;
-			if (hrefValue.startsWith('http') || hrefValue.startsWith('https')) {
-				menu.append(new gui.MenuItem({
-					label: 'Open Link in Browser',
-					click: function () {
-						gui.Shell.openExternal(hrefValue);
-					}
-				}));
+
+			// Relative URL, let's prepend the team URL.
+			if (hrefValue.startsWith('/')) {
```

Comment:
Sometimes URLs can be `//slack.com/hi` which uses the same protocol as the existing page. I tried using `url.parse` to detect `hostname` but it looks like that won't work:

http://requirebin.com/?gist=86f3a3a987c9779dec0b

However, I wrote a module a while back for work that seems to line up with this use case. Maybe we should use that:

https://github.com/underdogio/resolve-link